src/events/cd-booking-create-account.vue

```diff
@@ -145,6 +145,7 @@
         return UserService.getCurrentUser().then((response) => {
           const user = response.body.user;
           const selectedEvent = StoreService.load('selected-event');
+          // TODO: swap initUserType with user type based off dob
```

Comment:
Needs to be done first.

Reply:
how is this related to join ?

Reply:
Because it joins as a specific user type. As the code is here, it'll join based off the init user type, which could be champion or mentor for legacy users. The idea here is to limit it to only parent-guardian/attendee-o13 based of the DoB of the logged in user.